cli/src/build/onboarding/ui/shell.tsx

```
@@ -0,0 +1,136 @@
+// --platform / a single native dir there's no picker to keep, so we briefly
+// show just the framed header until the app mounts — startup, not mid-flow.)
+//
+// `command.ts` passes `initialPlatform` to skip the picker, and
+// `onResolvePlatform` so it can print the post-exit completion breadcrumb.
+import { Box, useStdout } from 'ink'
+import React, { useCallback, useEffect, useState } from 'react'
+import { loadAndroidProgress } from '../android/progress.js'
+import AndroidOnboardingApp from '../android/ui/app.js'
+import { loadProgress } from '../progress.js'
+import OnboardingApp from './app.js'
+import { PICKER_MIN_COLS, PICKER_MIN_ROWS, terminalFitsPicker } from '../min-terminal-size.js'
```

ESLint

[error] Expected "../min-terminal-size.js" to come before "./app.js".

(perfectionist/sort-imports)